pkg/volume/volume_linux.go

```diff
@@ -33,20 +33,36 @@ const (
 	execMask = os.FileMode(0110)
 )
 
+// calculateMask starts with a read+write mask and opens them further as needed.
+func calculateMask(mounter Mounter, mode os.FileMode, dir bool) os.FileMode {
+	mask := rwMask
+	if mounter.GetAttributes().ReadOnly {
+		mask = roMask
+	}
+	if dir {
+		mask |= os.ModeSetgid
+		mask |= execMask
+	}
+	return mode | mask
```

Comment:
Document what this returns better. I hate pipes

Reply:
done

Reply:
, btw, no idea why github doesnt  show this but I added a comment above :)